src/libsyncengine/syncpal/syncpal.cpp

```diff
@@ -964,6 +964,10 @@ ExitCode SyncPal::updateSyncNode() {
 }
 
 std::shared_ptr<Snapshot> SyncPal::snapshot(ReplicaSide side, bool copy) {
+    if (side == ReplicaSideUnknown) {
+        LOG_ERROR(_logger, "Call to SyncPal::snapshot with 'ReplicaSideUnknown').");
```

Comment:
The three strings are distinct (function names). Are you considering removing the function name from the log since we already have the line information?